arch/wasm32/wasm.js

```diff
@@ -910,6 +918,10 @@ var syscall = (function() {
       print('syscall(' + n + ', ' + a + ', ' + b + ', ' + c + ', ' + d + ', ' +
             e + ', ' + f + ')');
       return -1; },
+    __syscall: function(n, a, b, c, d, e, f, g) {
```

Comment:
This seems incorrect?

Reply:
Hmm.. since __syscall is declared as varargs.  But it does need to exist.  It ends up being used for syscalles with 7 args since there is no __syscall7.   See ./src/internal/syscall.h.  For an example of a callsite see ./src/fcntl/posix_fadvise.c which ends up being a 7 arg call.